front50-oracle/src/main/java/com/netflix/spinnaker/front50/model/OracleStorageService.java

```diff
@@ -6,6 +6,8 @@
  * If a copy of the Apache License Version 2.0 was not distributed with this file,
  * You can obtain one at https://www.apache.org/licenses/LICENSE-2.0.html
  */
+// changed timeCreated() to timeModified() in code below to fix pipeline issue
+
```

Comment:
Please remove and let the changelog speak for itself. This comment's usefulness will quickly decay.

Reply:
Went ahead and removed those 2 lines and pushed back up the changes.